python/fi_instrumentation/fi_types.py

```
@@ -896,60 +897,130 @@ def __post_init__(self):
         if not isinstance(self.type, EvalTagType):
             raise ValueError(f"type must be an EvalTagType enum, got {type(self.type)}")
 
-        if not isinstance(self.eval_name, EvalName):
+        if not self.eval_name: 
             raise ValueError(
-                f"eval_name must be an EvalName enum, got {type(self.eval_name)}"
+                f"eval_name is required"
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)